yeat/workflow/Oxford

```diff
@@ -87,3 +87,20 @@ rule unicycler:
         unicycler -l {input.reads} -t {threads} -o {params.outdir} {params.extra_args}
         ln -s assembly.fasta {output.contigs}
         """
+
+
+rule metamdbg:
+    output:
+        contigs="analysis/{sample}/{readtype,nano-raw|nano-corr|nano-hq}/{label}/metamdbg/contigs.fasta"
+    input:
+        reads="seq/input/{sample}/{readtype}/combined-reads.fq.gz"
+    threads: 128
+    params:
+        outdir="analysis/{sample}/{readtype}/{label}/metamdbg",
+        extra_args=lambda wildcards: config["assemblies"][wildcards.label].extra_args
+    shell:
+        """
+        metaMDBG asm --out-dir {params.outdir} --in-ont {input.reads} -t {threads} {params.extra_args}
+        gunzip {params.outdir}/contigs.fasta.gz
+        """
```

Comment:
For nanopore reads, `metaMDBG` can take in various types of simplex Nanopore reads (R10.4+).

See v1.0 release notes.

 https://github.com/GaetanBenoitDev/metaMDBG/releases/tag/metaMDBG-1.0